gulpconfig.js

```diff
@@ -16,6 +16,17 @@ var options = {
         // Include paths to resolve automatically.
         includePaths: ['./bower_components/']
       }
+    },
+    {
```

Comment:
Makes a lot of sense, using partials and non-partials simplifies the set up considerably.